install/amazon_rds/00_overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Step 0: Overview'
+install_track_title: Installation Guide (Amazon RDS and Amazon Aurora)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+To monitor your Amazon RDS for PostgreSQL or Amazon Aurora PostgreSQL servers
+with pganalyze, you need to install the pganalyze collector on an Amazon EC2
+instance, with Amazon ECS, or with Docker.
+The collector running in your environment sends query data and other statistics
+on a continuous basis to pganalyze.
```

Comment:
The current wording is a little awkward, since it could be read like "on an EC2 instance... with ECS or with Docker" which doesn't really make sense. Breaking this out into a separate sentence makes it easier to follow, but my version could probably still be improved.

We should also avoid saying "query data", since users could get the mistaken impression that query results are sent to pganalyze.

```suggestion
To monitor your Amazon RDS for PostgreSQL or Amazon Aurora PostgreSQL servers
with pganalyze, you need to run the pganalyze collector to connect to your database
and send query metadata and other statistics on a continuous basis to pganalyze.
The collector can be installed directly on an Amazon EC2 instance, with Amazon ECS,
or with Docker.
```

Reply:
I'm going to merge as your version (to reduce the rebasing pain) but here is some other ideas:

"The collector can be installed directly on an Amazon EC2 instance or with Amazon ECS, depending on your operational environment."

Basically, I'm dropping Docker here. Docker is one of the options, but feels like it can be dropped from this summary (mainly to simplify and avoid the awkwardness that you feel). When we add EKS, maybe we can say something like:

"The collector can be installed directly on an Amazon EC2 instance, with Amazon ECS, or with Amazon EKS, depending on your operational environment."

Regarding the query data part, +1 on your change, though I think this can also be reworded some, like the following:

"To monitor your Amazon RDS for PostgreSQL or Amazon Aurora PostgreSQL servers with pganalyze, you need to run the pganalyze collector. The pganalyze collector connects to your database to capture query metadata and various statistics, then relays this information back to pganalyze on a continuous basis."

I split this to two sentences, also add the description that the collector capture these data too. What do you think?

Reply:
Yes. +1 on the second suggestion, especially. One of the "cheat codes" for clearer writing is shorter sentences. And your version flows better, since my sentence just assumes you will infer where the collector gathers these stats.